samples/matter/light_bulb/src/app_task.cpp

```diff
@@ -207,6 +231,28 @@ CHIP_ERROR AppTask::StartApp()
 	return CHIP_NO_ERROR;
 }
 
+#if NUMBER_OF_BUTTONS == 2
+void AppTask::StartBLEAdvertisementAndLightActionEventHandler(const AppEvent &event)
+{
+	if (event.ButtonEvent.Action == static_cast<uint8_t>(AppEventType::ButtonPushed)) {
```

Comment:
I know this may be copied from other samples, but why do we use `AppEventType` to encode `pushed/released` while it's not actually the event type, but rather additional info associated with `Button` type? We could avoid this static_cast and provide better type safety (which can be a followup task, I guess)

Reply:
It was to aggregate all event-like entities in one place. But ok, pushed/release are not really pure events. I may create a ticket to change it,